vcpkg-artifacts/test/sequence-equal.ts

```diff
@@ -3,14 +3,12 @@
 
 import { fail, strict } from 'assert';
 
-// I like my collections to be easier to compare.
```

Comment:
I like my collection comparisons to not require funny things like

```
// forces the global function for sequence equal to be added to strict before this exectues:
s;
```

or adding functions to prototypes we don't own like `assert` and `strict`.